### ISSUE TITLE
Sim10k dataset

### ISSUE THREAD

**Issue by Re-dot-art** [justinkay/aldi#19]
Hello, could you please provide the partitioning results of the training and testing sets for the sim10k dataset? The dataset I downloaded only includes the training set. thanks

**Comment by justinkay** [comment]
Hi @Re-dot-art , we use the same settings for Sim10k -> Cityscapes as other works, which don't use a Sim10k validation set. We and others validate directly on Cityscapes (bad practice, as noted in the conclusion of our paper). Note for this task we also only evaluate on the 'cars' class like prior work, so make sure you get the [right Cityscapes test labels](https://github.com/justinkay/aldi/releases/download/v0.0.1/cityscapes_val_instances_cars.json).

**Comment by Vigneshwar73** [comment]
Hi @justinkay, Wouldn't using cityscapes as validation set prove to counter intuitive for source-only model training and lead to negative learning of the model as the training set and validation set are from different domain?

**Comment by justinkay** [comment]
In general, performing model selection on the target dataset does seem to be beneficial (even though it really ought to be considered “cheating” by the UDA community). But it’s certainly possible this is not the optimal model selection strategy for burn-in. Would be curious to hear if you find a better method!
